### PR TITLE
Migrate tests/api/admin to fixtures

### PR DESCRIPTION
**Pull request by pcustic**
## Description

This is the continuation of the effort (started by @io7m) to move CM tests to use fixtures instead of class inheritance. 

## Motivation and Context

Motivation is to move all the tests to use fixtures.
[Notion](https://www.notion.so/lyrasis/Refactor-Admin-API-test-suite-4cb00937f47840c88450f3bb880cdfaf)

More can be read in previous PRs and tickets. 
[Here](https://github.com/ThePalaceProject/circulation/pull/637) and [here](https://github.com/ThePalaceProject/circulation/pull/751)

## How Has This Been Tested?

Ran all changed unit tests.

## Checklist

- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.

### CODE REVIEW

**Review by RishiDiwanTT — COMMENTED** [review]
Some very minor comments, otherwise all looks good. :+1:

File: tests/api/admin/controller/test_admin_search_controller.py [hunk 1]
```diff
@@ -1,79 +1,91 @@
...
+
+class TestAdminSearchController:
+    def test_search_field_values(self, admin_search_fixture: AdminSearchFixture):
+        with admin_search_fixture.admin_ctrl_fixture.request_context_with_library_and_admin(
+            "/",
+            library=admin_search_fixture.admin_ctrl_fixture.ctrl.db.default_library(),
         ):
-            response = self.manager.admin_search_controller.search_field_values()
+            response = (
+                admin_search_fixture.admin_ctrl_fixture.ctrl.manager.admin_search_controller.search_field_values()
```
[the remark this answers is not in the document]
Comment (pcustic) [reply tests/api/admin/controller/test_admin_search_controller.py:87]: Yeah, some of these became a bit confusing, I'll try to simplify it 👍

File: api/controller.py [hunk 2]
```diff
@@ -135,6 +135,7 @@ class CirculationManager:
     log = logging.getLogger("api.controller.CirculationManager")
 
     def __init__(self, _db, testing=False):
+        self.admin_analytics_services_controller = None
```
[the remark this answers is not in the document]
Comment (pcustic) [reply api/controller.py:138]: I'm not sure 😂 Maybe something for the mypy, I'll check. This PR got dragged a bit so I forgot about half of these changes.

[the remark this answers is not in the document]
Comment (pcustic) [reply api/controller.py:138]: Leftover from development, removed.

File: tests/api/admin/controller/test_admin_search_controller.py [hunk 3]
```diff
@@ -1,79 +1,91 @@
...
-                w.presentation_edition.primary_identifier, s, gutenberg
-            )
+            db.classification(w.presentation_edition.primary_identifier, s, gutenberg)
+
 
-    def test_search_field_values(self):
-        with self.request_context_with_library_and_admin(
-            "/", library=self._default_library
+@pytest.fixture(scope="function")
+def admin_search_fixture(admin_ctrl_fixture) -> AdminSearchFixture:
```
[the remark this answers is not in the document]
Comment (pcustic) [reply tests/api/admin/controller/test_admin_search_controller.py]: Added https://github.com/ThePalaceProject/circulation/pull/913/commits/97facf6a2b00a75b149b11a128050e72c4886ce6

File: api/controller.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (pcustic) [reply api/controller.py]: https://github.com/ThePalaceProject/circulation/pull/913/commits/97facf6a2b00a75b149b11a128050e72c4886ce6

**Review by io7m — APPROVED** [review]
This seems good to me!

**Pull request merged**